qcodes/instrument_drivers/rohde_schwarz/RTO1000.py

```diff
@@ -367,6 +508,18 @@ def __init__(self, name: str, address: str,
                                         'CH4': 'CHAN4',
                                         'EXT': 'EXT'})
 
+        self.add_parameter('trigger_mode',
+                           label='Trigger mode',
+                           set_cmd='TRIGger:MODE {}',
+                           get_cmd='TRIGger1:SOURce?',
+                           val_mapping={'AUTO': 'AUTO',
+                                        'NORMAL': 'NORMal',
```

Comment:
Are you sure the command argument is case sensitive? If not, there is no need for a value mapping

Reply:
I've tested on the RTO1024 and the commands are not case sensitive. So change the mapping to a vals so int inputs are validated.